src/main/java/org/jabref/logic/importer/fileformat/PdfMergeMetadataImporter.java

```diff
@@ -136,6 +136,27 @@ public ParserResult importDatabase(Path filePath) throws IOException {
         return new ParserResult(List.of(entry));
     }
 
+    /**
+     * A modified version of {@link PdfMergeMetadataImporter#importDatabase(Path)}, but it
+     * relativizes the {@code filePath} if there are working directories before parsing it
+     * into {@link PdfMergeMetadataImporter#importDatabase(Path)}
+     * (Otherwise no path modification happens).
+     *
+     * @param filePath    The unrelativized {@code filePath}.
+     */
+    public ParserResult importDatabase(Path filePath, BibDatabaseContext context, FilePreferences filePreferences) throws IOException {
+        Objects.requireNonNull(context);
+        Objects.requireNonNull(filePreferences);
+
+        List<Path> directories = context.getFileDirectories(filePreferences);
+
+        if (!directories.isEmpty()) {
+            filePath = FileUtil.relativize(filePath, directories);
```

Comment:
Doesn't that method also have a check for emptyness? Then, this call can be moved out of the if and the jf be removed.

Reply:
I have just added a commit that does that